### PR TITLE
[15.0] [MIG] stock_quant_manual_assign

### PR DESCRIPTION
**Pull request by JasminSForgeFlow**
Standard Migration

@ForgeFlow

### CODE REVIEW

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM functional and technical review

File: stock_quant_manual_assign/__manifest__.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 {
     "name": "Stock - Manual Quant Assignment",
-    "version": "14.0.1.1.0",
+    "version": "15.0.1.1.0",
```
Comment (AaronHForgeFlow) [inline stock_quant_manual_assign/__manifest__.py]: 15.0.1.0.0

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM

**Pull request merged**